<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_smartMaya</class>
 <widget class="QMainWindow" name="MainWindow_smartMaya">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ==============================================
                QWidget
=============================================== */
QWidget
{
                font:10pt &quot;Arial&quot;;
}
/* ==============================================
                QLineEdit
=============================================== */
QLineEdit
{
                padding: 1px;
                border: 1px solid #ffaa00;
                border-radius: 2px;
}
 
QLineEdit::hover
{
                border: 1px solid #55aaff;
}
 
/* ==============================================
                QPushButton
=============================================== */
QPushButton
{
                padding: 1px;
                border: 1px solid #ffaa00;
                border-radius: 2px;        
 
}
 
QPushButton::menu-indicator:open
{
                position: relative;          
                top: 2px; left: 2px;
}
QPushButton::hover
{
                border: 1px solid #55aaff;
}
/* ==============================================
                QLabel
=============================================== */
QLabel
{
                padding: 1px;
                border: 1px solid #ffaa00;
                border-radius: 2px;
}
 
QLabel::hover
{
                border: 1px solid #55aaff;
}
/* ==============================================
                QTreeWidget
=============================================== */
QTreeWidget
{
                padding: 1px;
                border: 1px solid #ffaa00;
                border-radius: 2px;
}
 
QTreeWidget::hover
{
                border: 1px solid #55aaff;
}
/* ==============================================
                QGroupBox
=============================================== */
QGroupBox
{
                padding: 1px;
                border: 1px solid #ffaa00;
                border-radius: 2px;
}
 
QGroupBox::hover
{
                border: 1px solid #55aaff;
}
/* ==============================================
                QProgressBar
=============================================== */
QProgressBar::hover
{
                border: 1px solid #55aaff;
}
/* ==============================================
                QTextEdit
=============================================== */
QTextEdit
{
                padding: 1px;
                border: 1px solid #ffaa00;
                border-radius: 2px;
}
 
QLabel::hover
{
                border: 1px solid #55aaff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget_smartMaya">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_toolBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_toolBar">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>5</number>
          </property>
          <widget class="QTextEdit" name="textEdit_result_2">
           <property name="lineWrapMode">
            <enum>QTextEdit::WidgetWidth</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="overwriteMode">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEdit_result_3">
           <property name="lineWrapMode">
            <enum>QTextEdit::WidgetWidth</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="overwriteMode">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="textEdit_result">
       <property name="lineWrapMode">
        <enum>QTextEdit::WidgetWidth</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="overwriteMode">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>15</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>subing85@gmail.com</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_importMayaFile"/>
    <addaction name="separator"/>
    <addaction name="action_importMelPython"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="action_preference"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_aboutApplication"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="action_startToExecute"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="action_importMayaFile">
   <property name="text">
    <string>Import Maya File</string>
   </property>
  </action>
  <action name="action_importMelPython">
   <property name="text">
    <string>Import Mel/Python</string>
   </property>
  </action>
  <action name="action_aboutApplication">
   <property name="text">
    <string>About Application</string>
   </property>
  </action>
  <action name="action_preference">
   <property name="text">
    <string>Preference</string>
   </property>
  </action>
  <action name="action_startToExecute">
   <property name="text">
    <string>Start To Execute</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
